presto-spi/src/main/java/com/facebook/presto/spi/statistics/ColumnStatistics.java

```diff
@@ -34,11 +34,11 @@ private ColumnStatistics(Estimate dataSize, Estimate nullsCount, Estimate distin
         statistics = createStatisticsMap(dataSize, nullsCount, distinctValuesCount);
     }
 
-    private static Map<String, Estimate> createStatisticsMap(Estimate dataSize, Estimate nullsCount, Estimate distinctValuesCount)
+    private static Map<String, Estimate> createStatisticsMap(Estimate dataSize, Estimate nullsFraction, Estimate distinctValuesCount)
     {
         Map<String, Estimate> statistics = new HashMap<>();
         statistics.put(DATA_SIZE_STATISTIC_KEY, dataSize);
-        statistics.put(NULLS_COUNT_STATISTIC_KEY, nullsCount);
+        statistics.put(NULLS_FRACTION_STATISTIC_KEY, nullsFraction);
```

Comment:
i know no change here, can any of values being put be null? Maybe in such a case it's better not to put such a value at all? Otherwise annotate params with `@Nullable` (here and in `ColumnStatistics` ctor). Oh, in any case, annotate the getters as `@Nullable`.

(i see now you changed this not to be a map later on, but most of the comment still holds)